pkgs/flox/default.nix

```diff
@@ -97,15 +99,24 @@ in
           --bash <($out/bin/flox --bpaf-complete-style-bash) \
           --fish <($out/bin/flox --bpaf-complete-style-fish) \
           --zsh <($out/bin/flox --bpaf-complete-style-zsh)
+
+        wrapProgram "$out/bin/flox"                                       \
+                    --prefix PATH :                                       \
+                      "${flox-bash}/bin:${flox-bash}/libexec:${git}/bin"
```

Comment:
we already run git only as `$GIT_BIN` and flox bash as `$FLOX_BASH` inside of flox, adding them to the PATH as well seems a bit counterintuitive to me